### ISSUE TITLE
Not Responding to Team Names

### ISSUE THREAD

**Issue by The-Dreamz** [CaptainMoustache/basebot#8]
Still lovin' the bot but since we've started using it this year - it's not responding to score commands. It either brings nothing back or says the team can't be found.

**Comment by CaptainMoustache** [comment]
The score command is working for all commands I tried. Can you tell me which command sequences aren't working for you?

**Comment by matt-lucem** [comment]
I'm seeing the same kind of issue, but only affecting team names with a space in them.

```
> basebot score red sox
Sorry, I could not figure out the date "sox"

> basebot score redsox
Hmmm I couldn't find and teams using 'redsox'
I couldn't find a team with the name redsox. Please try again.

> basebot score boston
Final score from 04/13/2022
(score is displayed)

> basebot score yankees
Final score from 04/13/2022
(score is displayed)

> basebot score blue jays
Sorry, I could not figure out the date "jays"
```

**Comment by mlemanski** [comment]
I'm seeing the same kind of issue, but only affecting team names with a space in them.

```
> basebot score red sox
Sorry, I could not figure out the date "sox"

> basebot score redsox
Hmmm I couldn't find and teams using 'redsox'
I couldn't find a team with the name redsox. Please try again.

> basebot score boston
Final score from 04/13/2022
(score is displayed)

> basebot score yankees
Final score from 04/13/2022
(score is displayed)

> basebot score blue jays
Sorry, I could not figure out the date "jays"
```

**Comment by CaptainMoustache** [comment]
Thanks for the detailed info, I'll look into it

**Comment by CaptainMoustache** [comment]
Fixed in e13765c4f55e8193e2e581e177b09bfeb92d3148 The score command will now take multipart searches
